massdriver.yaml

```diff
@@ -32,49 +32,141 @@ NodeGroup: &node_group
       default: 10
       minimum: 1
       maximum: 1000
-    node_size:
+    compute_type:
```

Comment:
Is this change going to break existing AKS deployment configurations?

Reply:
I tested this in my org and had no breaking issues. It shouldn't break because we're still using `node_size`, it just has different options based on the `compute_type`.

Reply:
But the `node_size` schema is now dependent on the `compute_type`, which will be empty for all existing clusters. Not sure what this will do.

Reply:
Tested how this interaction would look for a customer. They would need to go into the config and select the `compute_type` and redeploy, but should not experience any downtime (even if selecting a different type/size)

![computetype2](https://github.com/massdriver-cloud/azure-aks-cluster/assets/8037512/229f7478-e575-4f76-8600-d6d36a81cc86)